bentoml/server/bento_api_server.py

```diff
@@ -288,6 +288,41 @@ def setup_instruments(self):
             labelnames=['api_name', 'service_version'],
         )
 
+        def before_request():
+            self.request_start_time = default_timer()
+            api_name = _request_ctx_stack.top.request.url_rule.endpoint
```

Comment:
I thought you can just do `api_name = request.endpoint` here?

Reply:
Ok, will update.